src/pages/index.js

```diff
@@ -1,11 +1,14 @@
+import Api from "../components/Api.js";
```

Comment:
Needs correction: Please don't reset the forms right after the submission, you should reset them only after the successful api response

Reply:
I have corrected this, thank you